examples/05_stable_diffusion/src/modeling/unet_2d_condition.py

```diff
@@ -109,10 +163,10 @@ def __init__(
             input_channel = output_channel
             output_channel = block_out_channels[i]
             is_final_block = i == len(block_out_channels) - 1
-
             down_block = get_down_block(
                 down_block_type,
                 num_layers=layers_per_block,
+                transformer_layers_per_block=transformer_layers_per_block[i],
```

Comment:
See [here](https://github.com/facebookincubator/AITemplate/blob/d6730854630d153a7ed3f88305a30852eba35ad9/examples/05_stable_diffusion/src/compile_lib/compile_unet_alt.py#L146-L149). `transformer_layers_per_block` is new for XL, v1 configs do not contain it, and the default is 1.
```
>>> from diffusers import UNet2DConditionModel
>>> v1 = UNet2DConditionModel.from_pretrained("runwayml/stable-diffusion-v1-5", subfolder="unet")
>>> xl = UNet2DConditionModel.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", subfolder="unet")
>>> v1.config.transformer_layers_per_block
1
>>> xl.config.transformer_layers_per_block
[1, 2, 10]
```
I've updated the default value to what is actually passed, `[1, 1, 1, 1]` to avoid confusion.